src/huggingface_hub/commands/user.py

```diff
@@ -164,11 +164,19 @@ def run(self):
         _|    _|  _|    _|  _|    _|  _|    _|    _|    _|    _|_|  _|    _|      _|        _|    _|  _|        _|
         _|    _|    _|_|      _|_|_|    _|_|_|  _|_|_|  _|      _|    _|_|_|      _|        _|    _|    _|_|_|  _|_|_|_|
 
+        To login, `huggingface_hub` now requires a token generated from https://huggingface.co/settings/token.
+        To login with username and password instead, interrupt with Ctrl+C.
         """
         )
-        username = input("Username: ")
-        password = getpass()
-        _login(self._api, username, password)
+
+        try:
+            token = getpass("Token: ")
+            _login(self._api, token=token)
+
+        except KeyboardInterrupt:
+            username = input("\rUsername: ")
+            password = getpass()
+            _login(self._api, username, password)
```

Comment:
do we really need to still support this? what's the timeline to remove it?

Reply:
in my opinion we could already remove it

Reply:
I think we need to offer the same API for a bit, but we could indicate clearly it's deprecated and will be removed in n months.

Reply:
I'll leave it like this for this release (0.2.0) with a deprecation warning, but will remove as soon as it's released, if that works for you.

Reply:
yes ok it works for me